site/src/components/sections/SectionInfo.vue

```diff
@@ -2,39 +2,255 @@
   <div>
     <b-modal :id="'section-info' + section.crn" :title="modalTitle">
       <div class="font-weight-bold">Prerequisites:</div>
-      <span v-html="formatPrerequisites(section.crn) || 'None'"></span>
+      <ul v-html="formatPrerequisites(section.crn) || 'None'"></ul>
```

Comment:
Why was this changed from a `span` to a `ul`?

Reply:
This was changed for consistency. Notice that in the example screenshot, the prerequisites are indented in the same way that the cross-listing list and major restriction list are; this is what using <ul> instead of <span> does. This change can be dropped without any issues; I just thought it looked better.

Reply:
I think we should keep it as a span since it's not rendering a list.